### DISCUSSION TITLE
Some questions about isar

### DISCUSSION THREAD

**Discussion by water-user** [isar/isar#584] [Help]
I read all the isar documentation (https://isar.dev/tutorials/quickstart.html) i have some questions:

About Filter and Where section, i see the where clauses is better: "Where clauses have two more superpowers: They give you "free" sorting and a super fast distinct operation."
So why not use only where clauses?

About "Watching Collections"/"watchLazy", I understand if some object added to db and notify me, so I need to rebuild all of my list again because i can't really know which object is added.
There is a way to get notify when some object is add, and fetch only the specific added object?

About Pagination with isar, can you show any code how to implement it?

"There is only one very important thing to keep in mind: You may change the converter for an existing field but the database type MUST BE THE SAME. Otherwise the schema migration will fail."
Not really understand what you mean, can you add code example?

Thanks!!

**Comment by maxiloEmmmm** [comment]
@simc any plan? provide some info(about change item) when collection change
